posthog/temporal/workflows/snowflake_batch_export.py

```diff
@@ -135,7 +180,7 @@ async def insert_into_snowflake_activity(inputs: SnowflakeInsertInputs):
                 interval_end=inputs.data_interval_end,
             )
 
-            local_results_file = tempfile.NamedTemporaryFile()
+            local_results_file = tempfile.NamedTemporaryFile(suffix=".jsonl")
```

Comment:
Sorry about the awkwardness of the file chunking here, feel free to rip it out and replace it with something better

Reply:
I'm thinking of changing bits around, but focusing on getting errors first...